pkg/devfile/parser/parse.go

```diff
@@ -441,6 +456,15 @@ func getDevfileFromRegistry(id, registryURL, version string) ([]byte, error) {
 	return util.HTTPGetRequest(param, 0)
 }
 
+func getStackFromRegistry(id, registryURL, destDir string) {
```

Comment:
this function should return error, do not ignore those errors. tools and users need to know if library failed to get the parent.